Library/Notifications.swift

```diff
@@ -1,5 +1,13 @@
+import Foundation
+
 public enum CurrentUserNotifications {
   public static let sessionStarted = "CurrentUserNotifications.sessionStarted"
   public static let sessionEnded = "CurrentUserNotifications.sessionEnded"
   public static let userUpdated = "CurrentUserNotifications.userUpdated"
 }
+
+extension Notification.Name {
+  public static let ksr_sessionStarted = Notification.Name(rawValue: CurrentUserNotifications.sessionStarted)
+  public static let ksr_sessionEnded = Notification.Name(rawValue: CurrentUserNotifications.sessionEnded)
+  public static let ksr_userUpdated = Notification.Name(rawValue: CurrentUserNotifications.userUpdated)
```

Comment:
now that notification names are a proper type, i wanna ween us off the stringy notifications and use things like this